sdk/keyvault/mgmt/src/main/java/com/azure/resourcemanager/keyvault/models/Sku.java

```diff
@@ -27,17 +27,12 @@
     @JsonProperty(value = "name", required = true)
     private SkuName name;
 
-    /** Creates an instance of Sku class. */
-    public Sku() {
-        family = "A";
-    }
```

Comment:
Track1 is constant.

Swagger def is modelAsString=true.
https://github.com/Azure/azure-rest-api-specs/blob/master/specification/keyvault/resource-manager/Microsoft.KeyVault/stable/2018-02-14/keyvault.json#L920-L930

Not sure what changed.